code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1059,19 +1059,19 @@
 		makeSkeleton()
 		return //No puking over skeletons, they don't smell at all!
 
-	if(!isturf(loc))
-		return
 
 	for(var/mob/living/carbon/human/H in range(decaylevel, src))
+		if(istype(loc,/obj/item/bodybag))

```

Comment:
There's a reason all of this was behind the prob() check- It keeps down on processing costs. Also, this istype needs a space in it.